packages/running/src/index.tsx

```diff
@@ -286,17 +269,15 @@ function Item(props: {
               tooltip={shutdownLabel}
             />
```

Comment:
We can also do better by adding `slot="end"`. But this requires to add that new property to the `ToolbarButtonComponent` properties in packages/ui-components/src/components/toolbar.tsx#L803